src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -125,29 +126,49 @@ struct ObjectPageDesc
-    { STR_OBJECT_SELECTION_FOOTPATHS,                   SPR_G2_LEGACY_PATH_TAB,      true  },
+    { STR_OBJECT_SELECTION_FOOTPATH_SURFACES,           SPR_G2_PATH_SURFACE_TAB,    false },
+    { STR_OBJECT_SELECTION_FOOTPATH_RAILINGS,           SPR_G2_PATH_RAILINGS_TAB,   false },
+    { STR_OBJECT_SELECTION_FOOTPATHS,                   SPR_G2_LEGACY_PATH_TAB,     true  },
     { STR_OBJECT_SELECTION_PATH_EXTRAS,                 SPR_TAB_SCENERY_PATH_ITEMS, false },
-    { STR_OBJECT_SELECTION_SCENERY_GROUPS,              SPR_TAB_SCENERY_STATUES,    false },
-    { STR_OBJECT_SELECTION_PARK_ENTRANCE,               SPR_TAB_PARK,               false },
+    { STR_OBJECT_SELECTION_PATH_SIGNS,                  SPR_TAB_SCENERY_SIGNAGE,    true  },
     { STR_OBJECT_SELECTION_WATER,                       SPR_TAB_WATER,              false },
-
-    // Dummy place holder for string objects
-    { STR_NONE,                   static_cast<uint32_t>(SPR_NONE),                  false },
-
     { STR_OBJECT_SELECTION_TERRAIN_SURFACES,            SPR_G2_TAB_LAND,            true  },
-    { STR_OBJECT_SELECTION_TERRAIN_EDGES,               SPR_G2_TERRAIN_EDGE_TAB,            true  },
-    { STR_OBJECT_SELECTION_STATIONS,                    SPR_G2_RIDE_STATION_TAB,               true  },
+    { STR_OBJECT_SELECTION_TERRAIN_EDGES,               SPR_G2_TERRAIN_EDGE_TAB,    true  },
+    { STR_OBJECT_SELECTION_PARK_ENTRANCE,               SPR_TAB_PARK,               false },
+    { STR_OBJECT_SELECTION_STATIONS,                    SPR_G2_RIDE_STATION_TAB,    true  },
     { STR_OBJECT_SELECTION_MUSIC,                       SPR_TAB_MUSIC_0,            false },
-    { STR_OBJECT_SELECTION_FOOTPATH_SURFACES,           SPR_G2_PATH_SURFACE_TAB,      false },
-    { STR_OBJECT_SELECTION_FOOTPATH_RAILINGS,           SPR_G2_PATH_RAILINGS_TAB,   false },
+    // Dummy place holder for string objects
+    { STR_NONE,                   static_cast<uint32_t>(SPR_NONE),                  false },
 };
 
+// Order of which the contents of each tab is displayed.
+ ObjectType static TabOrder[] = {
+                         ObjectType::Ride,
+                         ObjectType::SceneryGroup,
+                         ObjectType::SmallScenery,
+                         ObjectType::LargeScenery,
+                         ObjectType::Walls,
+                         ObjectType::FootpathSurface,
+                         ObjectType::FootpathRailings,
+                         ObjectType::Paths,
+                         ObjectType::PathBits,
+                         ObjectType::Banners,
+                         ObjectType::Water,
+                         ObjectType::TerrainSurface,
+                         ObjectType::TerrainEdge,
+                         ObjectType::ParkEntrance,
+                         ObjectType::Station,
+                         ObjectType::Music,
+                         ObjectType::ScenarioText
```

Comment:
Is the scenario text object type also a page in the object selection?

Reply:
It's not as far as I know, but it's listed in every list so I thought of still including it just to be certain.

Reply:
It’s not necessary any more, so I removed it for you.